chia/rpc/wallet_request_types.py

```diff
@@ -42,6 +42,109 @@ def default_raise() -> Any:  # pragma: no cover
     raise RuntimeError("This should be impossible to hit and is just for < 3.10 compatibility")
 
 
+@streamable
```

Comment:
Feel free to close this after reading.  I think there's room at some point to clearly define the request/response pairing in a programmatic way.